decentraland/unity-renderer
# Fix: skybox controller double instance

## What does this PR change?

Fixes: Fixed double instance issue of skybox controller. Skybox controller with flag feature is removed. Now procedural skybox will be default skybox.


## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer=https://glob-dland-s3-unity-renderer.s3.amazonaws.com/branch/fix/skybox-controller-double-instance
2. Check if skybox is moving and all features of procedural skybox is working from settings panel.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
